tella-locking-ui/src/main/java/com/hzontal/tella_locking_ui/ui/pin/pinview/OnKeyBoardClickListener.kt

```diff
@@ -34,7 +34,7 @@ class OnKeyBoardClickListener (private val minLength : Int,private val mPinLockL
         mPinLockListener?.onPinChange(mPin.length, mPin)
     }
```

Comment:
All the Calculator classes should be moved to the shared ui module, including the `KeyboardView` custom view